scania-digital-design-system/sdds
# docs(foundations): hid the code by default for all stories under Foundations

**Describe pull-request**  
Hid the code by default for all stories under Foundations, except Icon.

**Solving issue**  
Fixes: [AB#2728](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/2728)

**How to test**  
1. Go to storybook link below.
2. Check in Foundations -> Colour, Grid, Spacing and Typography
3. Check that the code is hidden by default.

**Screenshots**  
-

**Additional context**  
-
